initramfs/init.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/busybox sh
+#
+# SPDX-License-Identifier: AGPL-3.0+
+
+set -eu
+
+ROOT_MOUNT="/mnt/root"
+BB_BIN="/bin/busybox"
+
+init="/sbin/init"
+ro=""
+root=""
+rootflags=""
+rootfstype="auto"
+
+shutdown()
+{
+	while [ 1 ]; do
+		poweroff
+		echo "Please remove power to complete shutdown."
+		sleep 10s
+	done
+}
+
+restart()
+{
+	echo "Rebooting in 5 seconds ..."
+	sleep 5s
+	reboot
+	shutdown
+}
+rescue_shell()
+{
+	${BB_BIN} echo -e "\n" \
+		"##################################################\n" \
+		"#                   ▄▄▄      ▄▄▄                 #\n" \
+		"#                   ███████▄▄███▄                #\n" \
+		"#                 ▄███ ▄▄▄   ▄▄▄▐█               #\n" \
+		"#              ▄██  ██ ▀▀▀   ▀▀▀▐█▄              #\n" \
+		"#              ███▄▄██ ▄▄▄▄▄▄▄▄▄▐██              #\n" \
+		"#               ██████          ▐█               #\n" \
+		"#            ▄████▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀█▄           #\n" \
+		"#          ████▐█ ▐████████████████▌▐██▄         #\n" \
+		"#        ▐███▌▄▄█ ▐████████████████▌▐█▄█         #\n" \
+		"#         ███████ ███████▀▀ ▀██████▌▐██▌         #\n" \
+		"#         ███████ █████████ ██▐████▌▐██▌         #\n" \
+		"#        ▄███████ ██████▄██████████▌▐█▀█         #\n" \
+		"#        ████  ▐█ ▐████████████████▌▐█ █▌        #\n" \
+		"#        ▀███ ███   ▀▀▀▀            ▐██▀         #\n" \
+		"#           ▀▀████▄▄████████████████▀            #\n" \
+		"#             ▐████████▀▀████▀▀▌  █▌             #\n" \
+		"#            ▄██████  █  ▐███▄▄█▄▄██             #\n" \
+		"#            ████████▀▀▀█▀████▀    ██            #\n" \
+		"#            █████████    ▐███     ██            #\n" \
+		"#              ▀███▄█▄▄▄▄▄███▀▀▀▀▀▀▀             #\n" \
+		"#                                                #\n" \
+		"# Oops, something bad must have happened.        #\n" \
+		"# Starting busybox rescue and recovery shell.    #\n" \
+		"# Tip: type help<enter> for available commands.  #\n" \
+		"#                                                #\n" \
+		"##################################################\n"
```

Comment:
maybe add some todo's here or even an empty function with a todo for implementing part 4 of EM-2396